applications/graphs/controllers.py

```diff
@@ -229,6 +229,11 @@ def update_graph(request, graph_id, name=None, is_public=None, graph_json=None,
 		if name is not None:
 			G.set_name(name)
 
+		db.remove_tags_by_graph_id(request.db_session, graph_id=graph_id)
```

Comment:
@bruce-wayne99 Why was this method introduced?

Reply:
Editing tags involves **adding new tags** and **removing old tags** simultaneously, 
For example:
Let a user removed a tag named **x** from the graph **G** and added a tag **y** to the graph. So, there is a need to delete a row from `GraphToTag` containing the `graph_id` equal to the graph id of **G** and `tag_id` equal to tag id of tag **x** and also addition of a row with `graph_id` equal to the graph id of **G** and `tag_id` equal to tag id of tag **y**.
Now both these operations can be carried out at once if we remove all the entries of graph **G** in `GraphToTag` table  and after that adding the final list of tags the user wants to the graph.

Reply:
@adbharadwaj  I have provided an explanation for the usage of `db.remove_tags_by_graph_id`. Please review.